packages/wrangler/src/__tests__/pages.test.ts

```diff
@@ -1202,6 +1207,29 @@ describe("pages", () => {
 							include: ["/hello"],
 							exclude: [],
 						});
+
+						// Make sure the routing config is valid json
+						expect(() =>
+							JSON.parse(generatedFilepathRoutingConfig)
+						).not.toThrowError();
+						// The actual shape doesn't matter that much since this
+						// is only used for display in Dash, but this snapshot
+						// is a good way to track unintentional changes.
+						expect(generatedFilepathRoutingConfig).toMatchInlineSnapshot(`
```

Comment:
tiny suggestion: We're already calling `JSON.parse(generatedFilepathRoutingConfig)` - I think we should store the result and perform a deep equal rather than matching inline snapshot. Matching the snapshot tests for things we don't necessarily care about, like whitespace and getting the appropriate escaping is burdensome

Reply:
Great callout! Will fix

Reply:
Great callout! Will fix